src/metabase/models/revision.clj

```diff
@@ -73,7 +74,10 @@
 
 (t2/define-before-insert :model/Revision
   [revision]
-  (assoc revision :timestamp :%now))
+  (cond-> revision
+    true                                                   (assoc :timestamp :%now)
+    (and (map? (:object revision))
```

Comment:
is it only meant for cards revision?

if so we should define it in `defmethod revision/serialize-instance :model/Card` instead